hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/SCMPipelineManager.java

```diff
@@ -200,6 +210,16 @@ public Pipeline getPipeline(PipelineID pipelineID)
     }
   }
 
+  @Override
+  public boolean containsPipeline(PipelineID pipelineID) {
```

Comment:
Instead of contains, we could have made the get return an Optiona<Pipeline>

Reply:
Since the API is "contains", I wanted to keep it true/false. Can I address this in the next patch?

Reply:
It's a minor nit, I am ok with it asis. More of a preference thing, returning Optional already sets the contract and reduces code dup.